### ISSUE TITLE
Upgrading to 9.7.1 breaks GCP based setup due to custom logger levels

### ISSUE THREAD

**Issue by keesvanbemmel** [directus/directus#12407]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues?q=is%3Aissue) that already describes my problem.

### Describe the Bug

After upgrading to Directus 9.7.1, Directus will not start anymore and error out with:
```
/src/node_modules/directus/dist/logger.js:46

    for (const el of loggerEnvConfig.levels.split(',')) {

                                            ^


TypeError: loggerEnvConfig.levels.split is not a function or its return value is not iterable

    at Object.<anonymous> (/src/node_modules/directus/dist/logger.js:46:45)

    at Module._compile (node:internal/modules/cjs/loader:1101:14)

    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)

    at Module.load (node:internal/modules/cjs/loader:981:32)

    at Function.Module._load (node:internal/modules/cjs/loader:822:12)

    at Module.require (node:internal/modules/cjs/loader:1005:19)

    at require (node:internal/modules/cjs/helpers:102:18)

    at Object.<anonymous> (/src/node_modules/directus/dist/cache.js:10:34)

    at Module._compile (node:internal/modules/cjs/loader:1101:14)

    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
```

I'm pretty sure it has something to do with your recent fix to split Logger Levels @azrikahar 

### To Reproduce

Add this to your ENV variables (needed for GCP based setups):
```
LOGGER_LEVELS="trace:DEBUG,debug:DEBUG,info:INFO,warn:WARNING,error:ERROR,fatal:CRITICAL"
```

And startup directus.

### Errors Shown

See above.

### What version of Directus are you using?

9.7.1

### What version of Node.js are you using?

16 lts

### What database are you using?

Postgres 13

### What browser are you using?

na

### What operating system are you using?

MacOS

### How are you deploying Directus?

Docker

**Comment by azrikahar** [comment]
This has been reported in #12339 and a duplicate of it. 

It was unintendedly affected by #12330, and my fix in #12342 is to resolve it rather than cause it! 😄

You can refer to this workaround for `v9.7.1` in the mean time: https://github.com/directus/directus/issues/12339#issuecomment-1077033484 (basically turning it into a string so that `.split` can work again)

**Comment by keesvanbemmel** [comment]
Sorry man, should have checked open issues!

**Comment by azrikahar** [comment]
no worries! The fix from my PR has not been released yet, hence we'll have to use the workaround for now 👍

**Comment by arifamir** [comment]
Is this fix released?

**Comment by azrikahar** [comment]
@arifamir yes #12342 is merged in the latest release `v9.8.0`.
